ports/kerbal/vcpkg.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "kerbal",
+  "version-semver": "2023.12.1",
```

```suggestion
  "version": "2023.12.1",
```

No clear instructions for `semver` found upstream, it is recommended to use `version` instead of `version-semver`.